odoo/addons/base/models/ir_ui_view.py

```diff
@@ -498,6 +498,10 @@ def _compute_defaults(self, values):
     @api.model_create_multi
     def create(self, vals_list):
         for values in vals_list:
+            if 'arch_db' in values and not values['arch_db']:
```

Comment:
thanks for fixing the bug 😃 
one thing I want to leave for discussion:
if we could use
`if 'arch_db' in values and 'arch_base' in values:`
which denotes a general ORM logic that a `value` from a compute/inverse function will overwrite the `value` in the parameter when create/write.
So we can ignore/drop the `value` in parameter if we know it will be recomputed.
What do you think @rco-odoo